.github/CONTRIBUTING.md

```diff
@@ -306,43 +306,68 @@ eg. on MacOS you can follow this instructions:
 #1 install brew (https://brew.sh)
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 
-#2 install node 16x using NVM (https://github.com/nvm-sh/nvm)
-nvm install v16.17.0`
-nvm alias default v16.17.0`
+# before next step uninstall node
+brew uninstall --ignore-dependencies node 
+brew uninstall --force node
+* delete .npm and .node folders from your root folder *

```

Comment:
I don't think we should include this because it might break other brew installations for users. It does not matter what nodes you have installed before because `nvm alias default v16.17.0` will make sure nvm installed node is the priority in PATH.